pygenometracks/tracks/LinksTrack.py

```diff
@@ -46,17 +53,25 @@ def __init__(self, *args, **kwarg):
         valid_intervals = 0
         interval_tree = {}
         line_number = 0
+        has_score = True
```

Comment:
general idea for later ... should we have functions that set the defaults? This way we can change default globally ...

Reply:
Yes I agree.

Reply:
See #119